.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-json
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: fix-encoding-pragma
+      - id: no-commit-to-branch
+      - id: pretty-format-json
+        exclude: Pipfile.lock
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+      - id: black
+  - repo: https://github.com/asottile/reorder_python_imports
```

Comment:
I like the idea but would prefer `isort` especially for the multiple imports.

Reply:
updated with isort